src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  width: 1200px;
+  background-color: #fff;
+  justify-content: space-between;
+
+  .img {
+    display: block;
+    margin-left: 50px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+  .nav {
+    margin-right: 50px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  a {
+    text-decoration: none;
+    color: black;
+  }
+
+  ul {
+    display: flex;
+    font-size: 12px;
+    list-style: none;
+    text-transform: uppercase;
+    font-weight: bold;
+  }
+  li:not(:last-child) {
+    margin-right: 20px;
+  }
+}
+
+.blue {
+  color: #00acdc;
+}
+
+a:hover {
+  color: #00acdc;
```

Comment:
The color value `#00acdc` should be replaced with the CSS variable for the blue color. Ensure consistency by using the variable throughout the CSS file.